Report : Batch Item Expiry Status

revant opened the pull request:
- Select From Date and To Date
- Lists Item Batches about to expire or already expired
- Expiry Status column shows number of days in which item/batch will expire

![batch_item_expiry_status](https://cloud.githubusercontent.com/assets/1040161/21381244/266924d0-c780-11e6-93d5-015b350929af.png)

revant committed:
Report : Batch Item Expiry Status

Select From Date and To Date
Lists Items about to expire or already expired
Expiry Status column shows number of days in which item/batch will expire

revant committed:
Merge branch 'develop' of https://github.com/frappe/erpnext into develop

rmehta commented on erpnext/stock/report/batch_item_expiry_status/batch_item_expiry_status.py:
```diff
@@ -0,0 +1,94 @@
+		expires_in_days = (exp_date - frappe.utils.datetime.date.today()).days
+
+		if expires_in_days > 0:
+			qty_dict.expiry_status = "Expires in {} day(s)".format(expires_in_days)
```
translatable ? `_(""Expires in {} day(s)")`

revant replied:
![columnrename](https://cloud.githubusercontent.com/assets/1040161/21382431/a0005e02-c786-11e6-82c4-787e5d4e4883.png)

How about renaming the column to (already translatable) "Expiry (In Days)" and data as plain int?

rmehta replied:
Cool that works better!

Btw there is a better way using `get_columns` using JSON 

https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/report/profitability_analysis/profitability_analysis.py#L121

revant committed:
Report - Batch Item Expiry Status

Changed column name `Expiry Status` to `Expiry (In Days)`
Changed column type from Data to Int

revant committed:
Merge branch 'develop' of https://github.com/frappe/erpnext into develop

mdwala commented:
Isn't available quantity per batch is most required field for all?
